addon/components/frost-radio-button.js

```diff
@@ -89,12 +91,10 @@ export default Component.extend(PropTypeMixin, {
 
   // == Functions ===============================================================
 
-  _createEvent (_event, _target) {
-    let event = $.Event(null, _event)
-    let target = $.clone(_target)
-    target.id = get(this, 'groupId')
-    event.target = target
-    return event
+  _modifyEvent (event, target) {
```

Comment:
so trying to understand the code here. The util used here is converting a native event into an Ember event? And then you are resetting the event target?